src/index.js

```diff
@@ -4,19 +4,34 @@ const parse = require('./parse.js')
 
 /**
  * Primary loader function
- * 
+ *
  * @param {string} content - Markdown file content
  */
 module.exports = function(content) {
     const callback = this.async()
     const options = Utils.getOptions(this)
 
+    const { htmlLoaderOptions } = (options || {});
+
+    let attributes
+
     parse(content, options)
-        // @todo we should probably just intercept images in the tree
-        // or recommend that the `html-loader` be chained
```

Comment:
As noted in my previous comment, I would keep the `@todo` unless it's really impossible or impractical for us to recommend chaining here.

Reply:
Agreed that it would be ideal to just leave this to webpack. As it stands, I couldn't figure out how to make this work, because this loader needs to intervene both before _and_ after `html-loader`. Currently the loader passes its output to `html-loader`, receives the result, and then appends the frontmatter export to that result. Would love to hear if there's a smarter way to do this!

Reply:
Indeed, which may have been why I did it that way in the first place. I'll do a little research on this front and get back to you.

In the short term I would keep the todo and then maybe the best next steps would be to merge this as is and create an issue where we discuss a more robust refactor that allows for chaining and supports frontmatter via `plugins` rather than the current approach.